source/api.rst

```diff
@@ -0,0 +1,612 @@
+PXC API Documentation
+=====================
+
+.. raw:: html
+
+   <style>
+   
+   .toggle {
+        background: none repeat scroll 0 0 #ffebcc;
+        padding: 12px;
+        line-height: 24px;
+        margin-bottom: 24px;
+    }
+   
+   .toggle .header {
+       display: block;
+       clear: both;
+       cursor: pointer;
+   }
+   
+   .toggle .header:after {
+       content: " ▶";
+   }
+   
+   .toggle .header.open:after {
+       content: " ▼";
+   }
+   </style>
+
+Percona Operator for Percona XtraDB Cluster provides an `aggregation-layer extension for the Kubernetes API <https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/>`_. Please refer to the
+`official Kubernetes API documentation <https://kubernetes.io/docs/reference/>`_ on the API access and usage details.
+The following subsections describe the Percona XtraDB Cluster API provided by the Operator.
+
+.. contents:: :local:
+
+Prerequisites
+-------------
+
+1. prepare
+
+   .. code-block:: yaml
+
+      # set correct API adress
+      KUBE_CLUSTER=$(kubectl config view --minify -o jsonpath='{.clusters[0].name}')
+      API_SERVER=$(kubectl config view -o jsonpath="{.clusters[?(@.name==\"$KUBE_CLUSTER\")].cluster.server}")
+
+      # create service account and get token
+      kubectl apply -f deploy/crd.yaml -f deploy/rbac.yaml
+      KUBE_TOKEN=$(kubectl get secret $(kubectl get serviceaccount percona-xtradb-cluster-operator -o jsonpath='{.secrets[0].name}' -n $NAMESPACE) -o jsonpath='{.data.token}' -n $NAMESPACE | base64 --decode )
+
+2. get a list of PXC clusters
+
+   .. code-block:: bash
+
+      # run "kubectl --v=9 get pxc" command find curl command in the output
```

Comment:
Kubectl Command:

```suggestion
      kubectl --v=9 get pxc
```

I think we can remove `--v=9` from the commands. It was just an example for developers how to get 'curl' ... @delgod  what do you think?

Reply:
have sense